src/serverlist.cpp

```diff
@@ -155,6 +155,16 @@ std::vector<ServerListSpec> deSerialize(const std::string &liststring)
 			server["address"] = tmp;
 			std::getline(stream, tmp);
 			server["port"] = tmp;
+			bool unique = true;
+			for (ServerListSpec &added : serverlist) {
```

Comment:
this should be const we don't  modify the list objects

Reply:
done